docker/run_tests/run_tests.sh

```diff
@@ -5,9 +5,9 @@ if ! [ -f completed_initial_run ]
 then
   echo 'Initialising mampf' &> >(tee -a /usr/src/app/log/initialisation.log)
   echo running: bundle exec rails db:migrate &> >(tee -a /usr/src/app/log/initialisation.log)
-  bundle exec rails db:migrate &> >(tee -a /usr/src/app/log/initialisation.log)
+  bundle exec rails db:migrate > /usr/src/app/log/initialisation.log
```

Comment:
There should be a `rails db:create` before, otherwise the migrations will fail (and they do fail, at least in my attempt to build the container).